src/debugger/SteppingManager.js

```diff
@@ -11,19 +11,16 @@
 
 import { BabelNode } from "babel-types";
 import type { DebugChannel } from "./channel/DebugChannel.js";
-import type { StepIntoData, StoppedReason } from "./types.js";
+import type { StoppedReason } from "./types.js";
 import invariant from "./../invariant.js";
-import { IsStatement } from "./../methods/is.js";
+import { StepperInto } from "./Stepper.js";
 
 export class SteppingManager {
   constructor(channel: DebugChannel) {
     this._channel = channel;
-    this._stepIntoData = {
-      prevStopData: undefined,
-    };
   }
   _channel: DebugChannel;
-  _stepIntoData: StepIntoData;
+  _stepInto: void | StepperInto;
```

Comment:
Eventually, you want Array<Stepper> to hold all steppers.

Reply:
Should the different kind of steppers be held separately? If so, there can only be one step into stepper at a time.